daru.gemspec

```diff
@@ -33,6 +33,7 @@ Gem::Specification.new do |spec|
   spec.required_ruby_version = '>= 2.1.0'
 
   spec.add_runtime_dependency 'backports'
+  spec.add_runtime_dependency 'daru-io'
```

Comment:
@zverok - daru-io and daru are both runtime_dependencies of each other, and that's creating an infinite loop in travis builds while bundle installing. :trollface:

Reply:
Yeah, already thought of that. I believe that daru should be _development_ dependency of `daru-io` (meaning since "daru 1.0" there is no sense in installing `daru-io` by itself, as it is installed with `daru`)

Reply:
Hmm, should v0.1.1 be released for daru-io with this dependency change?

Reply:
Probably so. At the same time when `daru-1.0.0-pre` would be released, probably?..

Reply:
Acknowledged. Let me just add `daru-io` to the Gemfile rather than then gemspec, so that it doesn't interfere with travis CI builds. Before `daru-1.0.0-pre` release, this can be changed after `daru-io-0.1.1` releases. Can you add this into the milestone?